packages/components/addon/components/hds/table/tr.hbs

```diff
@@ -3,5 +3,9 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 <tr class="hds-table__tr" ...attributes>
+  {{#if @isSelectable}}
+    <Hds::Table::ThSelectable @scope={{@scope}} />
```

Comment:
In order to know which scope to use here, you have to have the information where the row is injected, if it's in the `<thead>` or in the `<tbody>`. What I would probably do, is to pass some argument to the row to provide this information, and add conditional logic accordingly to decide which scope value to use (`"row"` or `"col"`). This argument can be then used also to determine the function to invoke when the checkbox is clicked (this depends on the implementation of the logic associated to the checkbox elements).